ui/components/admin.js

```diff
@@ -36,6 +36,7 @@ function Grant ({ id, userID, grants }) {
         <DeleteModal
           open={open}
           setOpen={setOpen}
+          actionBtnTitle='Revoke'
```

Comment:
nit: a better title could be `primaryButtonText` (i.e. primary vs secondary buttons)